backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxSparkPlanExecApi.scala

```diff
@@ -294,6 +294,19 @@ class VeloxSparkPlanExecApi extends SparkPlanExecApi {
     GenericExpressionTransformer(newSubstraitName, children, newExpr)
   }
 
+  override def genArrayInsertTransformer(
+      substraitExprName: String,
+      children: Seq[ExpressionTransformer],
+      original: Expression): ExpressionTransformer = {
+    children match {
+      case Seq(left, posExpr, right, _) if posExpr.original == Literal(1) =>
+        // transformer to array_prepend implement in velox
```

Comment:
Nit: remove this comment or rephrase it like: Transformer for array_prepend.  (follow community comment style: capitalize initial letter and end up with period).

Reply:
Update with `Transformer for array_prepend.`